experiments/ClimaEarth/components/land/climaland_helpers.jl

```diff
@@ -51,7 +51,8 @@ function make_land_domain(
     verttopology = CC.Topologies.IntervalTopology(vertmesh)
     vert_center_space = CC.Spaces.CenterFiniteDifferenceSpace(verttopology)
     subsurface_space = CC.Spaces.ExtrudedFiniteDifferenceSpace(atmos_boundary_space, vert_center_space)
-    space = (; surface = atmos_boundary_space, subsurface = subsurface_space)
+    subsurface_face_space = CC.Spaces.face_space(subsurface_space)
```

Comment:
```suggestion
    subsurface_face_space = CL.Domains.obtain_face_space(subsurface_space)
```
In land they use `obtain_face_space` - we should probably do the same to be consistent

Reply:
I opened a [PR](https://github.com/CliMA/ClimaLand.jl/pull/1134#pullrequestreview-2809989443) in ClimaLand that removes `obtain_face_space` and replaces it with `face_space`. I'm fine with keeping `obtain_face_space` if you see a reason to.

Reply:
Oh perfect! No need to keep that function in ClimaLand if it exists in ClimaCore